### ISSUE TITLE
vue/html-indent rule does not work correctly

### ISSUE THREAD

**Issue by danyadev** [vuejs/eslint-plugin-vue#1240]
**Tell us about your environment**	
* **ESLint version:** 7.4.0
* **eslint-plugin-vue version:** 7.0.0-alpha.9
* **Node version:**	12.18.2

**Please show your full configuration:**

```js
{
  root: true,

  parserOptions: {
    sourceType: 'module',
    ecmaVersion: 2019
  },

  env: {
    browser: true,
    node: true,
    es2020: true
  },

  plugins: ['import'],

  settings: {
    'import/resolver': 'webpack',
    'import/ignore': []
  },

  extends: [
    'plugin:vue/vue3-recommended'
  ],

  rules: {
    // no vue/html-indent rule added
  }
}
```

**What did you do?**	

```vue
<template>
  <div
    :prop="
      a &&
      (b || c)
    "
  >
    ...
  </div>
</template>

<script>
let a;
let b;
let c;

const x = a &&
(b || c);

console.log(x);

export default {
  setup: () => ({ a, b, c })
};
</script>
```

**What did you expect to happen?**
no errors

**What actually happened?**
```
C:\Users\danyadev\Desktop\vk-desktop\src\components\Test.vue
  5:1  warning  Expected indentation of 8 spaces but found 6 spaces  vue/html-indent
```

If you look at the variable x in the code, you will notice that the same expression is written there, but no errors are written for it.

**Comment by ota-meshi** [comment]
Thank you for this issue.

This is due to the difference in specifications between the `vue/html-indent` rule and the ESLint core `indent` rule.

`vue/html-indent` rule does not provide the same indentation as `indent` rule. If you want to apply the same indentation of `vue/html-indent` rule to `<script>`, turn on the `vue/script-indent` rule and turn off the `indent` rule.

You can also use [Prettier](https://prettier.io/) to apply consistent indentation (and format) including `<style>`.
